crates/database/src/orders.rs

```diff
@@ -551,13 +551,11 @@ pub fn user_orders<'a>(
         .fetch(ex)
 }
 
-pub fn solvable_orders(
-    ex: &mut PgConnection,
-    min_valid_to: i64,
-    min_surplus_fee_timestamp: DateTime<Utc>,
-) -> BoxStream<'_, Result<FullOrder, sqlx::Error>> {
-    #[rustfmt::skip]
-    const QUERY: &str = const_format::concatcp!(
+/// The base solvable orders query used in specialized queries. Parameterized by valid_to.
+///
+/// Does not take limit order surplus fee into account.
+#[rustfmt::skip]
+const SOLVABLE_ORDERS: &str = const_format::concatcp!(
```

Comment:
naming-nit: I would call this "open orders" maybe, since `(surplus_fee IS NOT NULL)` is required for it to be solvable.